src/testing/TestContext.hpp

```diff
@@ -166,6 +166,8 @@ class TestContext {
   /// wheather this context is valid or not
   bool invalid = false;
 
+  precice::utils::Parallel::Communicator comm = MPI_COMM_NULL;
```

Comment:
Could you please add a comment here? One might wonder what comm this is. It's the overall communicator, right?

Reply:
Done. It is the communicator of the current participant.
So, the one with the requested amount of ranks.